pkg/command/service/measure.go

```diff
@@ -147,20 +152,37 @@ kperf service measure --svc-perfix svc --range 1,200 --namespace ns --concurrenc
 					for j := range svcChannel {
 						if len(j) != 2 {
 							fmt.Printf("lack of service name or service namespace and skip")
+							currentMeasureResult.failCount++
+							workerMeasureResults[index] = currentMeasureResult
+							group.Done()
+							continue
 						}
 						svc := j[0]
 						svcNs := j[1]
 						svcIns, err := servingClient.Services(svcNs).Get(context.TODO(), svc, metav1.GetOptions{})
 						if err != nil {
 							fmt.Printf("failed to get Knative Service %s\n", err)
+							if strings.Contains(err.Error(), "not found") {
+								currentMeasureResult.notFoundCount++
+								fmt.Println(currentMeasureResult.notFoundCount)
```

Comment:
Just print the `notFoundCount`? Or is it a debug print?

Reply:
Thanks! fixed